tasks/eth/006-MCP-L1/SignFromJson.s.sol

```diff
@@ -0,0 +1,346 @@
+import {SystemConfig} from "@eth-optimism-bedrock/src/L1/SystemConfig.sol";
+import {Constants, ResourceMetering} from "@eth-optimism-bedrock/src/libraries/Constants.sol";
+import {L1StandardBridge} from "@eth-optimism-bedrock/src/L1/L1StandardBridge.sol";
+import {L2OutputOracle} from "@eth-optimism-bedrock/src/L1/L2OutputOracle.sol";
+import {ProtocolVersion, ProtocolVersions} from "@eth-optimism-bedrock/src/L1/ProtocolVersions.sol";
+import {SuperchainConfig} from "@eth-optimism-bedrock/src/L1/SuperchainConfig.sol";
+import {OptimismPortal} from "@eth-optimism-bedrock/src/L1/OptimismPortal.sol";
+import {L1CrossDomainMessenger} from "@eth-optimism-bedrock/src/L1/L1CrossDomainMessenger.sol";
+import {OptimismMintableERC20Factory} from "@eth-optimism-bedrock/src/universal/OptimismMintableERC20Factory.sol";
+import {L1ERC721Bridge} from "@eth-optimism-bedrock/src/L1/L1ERC721Bridge.sol";
+import {Predeploys} from "@eth-optimism-bedrock/src/libraries/Predeploys.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+
+contract SignFromJson is OriginalSignFromJson {
+    address constant finalSystemOwner = 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A;
+
+    address constant superchainConfigGuardian = 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A;
+
+    uint256 constant l2BlockTime = 2;
+
+    address constant p2pSequencerAddress = 0xAAAA45d9549EDA09E70937013520214382Ffc4A2;
+
+    address constant batchInboxAddress = 0xFF00000000000000000000000000000000000010;
+
+    address constant batchSenderAddress = 0x6887246668a3b87F54DeB3b94Ba47a6f63F32985;
+
+    uint256 constant l2OutputOracleSubmissionInterval = 1800;
+
+    uint256 constant l2OutputOracleStartingTimestamp = 1686068903;
+
+    uint256 constant l2OutputOracleStartingBlockNumber = 105235063;
+
+    address constant l2OutputOracleProposer = 0x473300df21D047806A082244b417f96b32f13A33;
+
+    address constant l2OutputOracleChallenger = 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A;
+
+    uint256 constant finalizationPeriodSeconds = 604800;
+
+    uint256 constant l2GenesisBlockGasLimit = 0x1c9c380;
+
+    uint256 constant gasPriceOracleOverhead = 188;
+
+    uint256 constant gasPriceOracleScalar = 684000;
+
+    uint256 constant systemConfigStartBlock = 17422444;
+
+    uint256 constant requiredProtocolVersion = 0x0000000000000000000000000000000000000003000000010000000000000000;
+
+    uint256 constant recommendedProtocolVersion = 0x0000000000000000000000000000000000000003000000010000000000000000;
+
+    Types.ContractSet proxies;
+
+    uint256 xdmSenderSlotNumber;
+
+    /// @notice Sets up the contract
+    function setUp() public {
+        proxies = _getContractSet();
+
+        // Read the slot number for the xDomainMsgSender in the L1CrossDomainMessenger
+        try vm.readFile(
+            string.concat(
```

Comment:
How do we feel about adding the monorepo as a dep here? cc @mds1 @maurelian 

Wondering if we could just include a hardcoded 204 and then a link to the code to validate. While reading it directly is really nice, its a super heavy solution

Reply:
I lean towards removing the dep and just using a hyperlink to allow for easy validation. Would greatly simplify things

Reply:
I'm ok with hardcoding and using a link to validate to keep it lighter weight, as if we have the wrong slot the assertion should fail. Here is the thread that led to using it as a dep, for context: https://github.com/ethereum-optimism/superchain-ops/pull/119#discussion_r1548249035